src/frontend/src/app/[locale]/(auth)/dashboard/layout.tsx

```diff
@@ -32,7 +32,7 @@ export default async function DashboardLayout(props: { children: React.ReactNode
   }
 
   return (
-    <>
+    <div className="flex h-screen flex-col overflow-hidden">
```

Comment:
Do we need this change    <div className="flex h-screen flex-col overflow-hidden">